storage/tokudb/ha_tokudb_admin.cc

```diff
@@ -114,10 +113,19 @@ recount_rows_t::recount_rows_t(
     }
 
     _throttle = tokudb::sysvars::analyze_throttle(thd);
+
+    snprintf(_parameters,
+             sizeof(_parameters),
+             "TOKUDB_ANALYZE_THROTTLE=%llu;",
+             _throttle);
+    _status[0] = '\0';
 }
 recount_rows_t::~recount_rows_t() {
 }
 void recount_rows_t::on_run() {
+    const char* orig_proc_info = NULL;
```

Comment:
Yes, it is correct, but it is also not correct :) This should be using the tokudb wrappers of tokudb_thd_get/set_proc_info that return and accept const char* pointers instead of the direch thd_* calls.